### PR TITLE
fix(BackToTop): use $accent-color. Fix #1121

### LINKED ISSUE #1121: Plugin-back-to-top doesn't respect theme color
- [x] I confirm that this is a issue rather than a question.

## Bug report
Changing $accentColor in palette.styl doesn't affect back-to-top arrow color.
#### Version
Alpha 30

#### Steps to reproduce
Change $accentColor in palette.styl. All site color update except the back to top arrow.

#### What is expected?
The arrow should use accentColor.

#### What is actually happening?
The hex is hardcoded in css (not even stylus)

#### Other relevant information

- Your OS: macOS 10.14.1
- Node.js version: 9.8.0
- Browser version: Chrome 72
- Is this a global or local install? global
- Which package manager did you use for the install? yarn

### PR DESCRIPTION
**Pull request by sh7dm**
**Summary**
Use `$accent-color`. Fixes #1121.

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: packages/@vuepress/plugin-back-to-top/BackToTop.vue [hunk 1]
```diff
@@ -59,19 +59,19 @@ export default {
...
-<style scoped>
+<style lang='stylus' scoped>
 .go-to-top {
   cursor: pointer;
   position: fixed;
   bottom: 2rem;
   right: 2.5rem;
   width: 2rem;
-  color: #72cda4;
+  color: $accentColor;
```
Comment (janvennemann) [inline packages/@vuepress/plugin-back-to-top/BackToTop.vue]: Maybe apply [lighten](http://stylus-lang.com/docs/bifs.html#lightencolor-amount) here to maintain the hover effect.

Comment (sh7dm) [reply packages/@vuepress/plugin-back-to-top/BackToTop.vue]:
```suggestion
.go-to-top:hover {
   color: lighten($accentColor, 30%);
 }
```

Comment (ulivz) [reply packages/@vuepress/plugin-back-to-top/BackToTop.vue]: +1

Comment (sh7dm) [reply packages/@vuepress/plugin-back-to-top/BackToTop.vue]: @janvennemann @ulivz fixed in a2af31f

**Pull request merged**